### ISSUE TITLE
Ruby 2.7 Deprecation Warnings

### ISSUE THREAD

**Issue by tpett** [DripEmail/drip-ruby#72]
Ruby 2.7 emits warnings for using the last argument of a method call implicitly as keyword arguments. It appears that `Drip::Client` does this on line 66 when initializing an instance of `Configuration` causing a deprecation warning output to the standard error stream.

**Deprecation Warnings Emitted**
```
drip-ruby-3.4.0/lib/drip/client.rb:66: warning: Using the last argument as keyword parameters is deprecated; maybe ** should be added to the call
drip-ruby-3.4.0/lib/drip/client/configuration.rb:13: warning: The called method `initialize' is defined here
```

The fix for this occurrence would be to simply prefix the argument with `**` though there may be other cases where this is a problem in the library that I have not encountered yet.

**Comment by will-in-wi** [comment]
Thanks for pointing this out! We'll take a look.

**Comment by will-in-wi** [comment]
This is fixed in version 3.4.1.
